### ISSUE TITLE
Tutorial at Gateways 2019

### ISSUE THREAD

**Issue by zonca** [jupyterlab/scipy2018-jupyterLab-tutorial#7]
Hi,
I'm co-chair of the Scientific Gateways 2019 conference in San Diego 23-25 September (https://sciencegateways.org/web/gateways2019).
Jupyter notebook and lab are already widespread in Scientific Gateways (websites for easy access to High Performance Computing resources) but often the gateway itself is implemented in some other web technology. But I think it would improve the user experience to have some functionality of the gateway (i.e. job submission / monitoring) built into JupyterLab as an extension.

Is there anyone available and willing to teach this tutorial (possibly 3 hours)? Possibly someone not too far away?

**Comment by Carreau** [comment]
@saulshanabrook @jasongrout @blink1073 @afshin @choldgraf ?
I might be I have to check with my employer.

**Comment by choldgraf** [comment]
We could also ask some of the people from LBNL and the folks that recently attended the HPC workshop. I think they've got quite a lot of experience in building science gateways using the jupyter ecosystem.

@zonca you might also want to post this in the [jupyter community forum](https://discourse.jupyter.org) since it cuts across a wider part of the Jupyter community

**Comment by saulshanabrook** [comment]
I won't be at SciPy so unfortunately I am unable to.

**Comment by jasongrout** [comment]
> Is there anyone available and willing to teach this tutorial (possibly 3 hours)?

What tutorial are you referring to? A generic JupyterLab tutorial? One that also covers writing new JupyterLab extensions? A rerun of our scipy tutorial? Or something more specific about setting up JupyterLab in the context of HPC?

**Comment by jasongrout** [comment]
> What tutorial are you referring to?

Sorry, I should have realized what repo this issue was on. So you mean the scipy tutorial? (we are updating it for scipy2019 too).

CC @ian-r-rose who is close, as well as @gnestor. I will look into if I'm available as well.

**Comment by jasongrout** [comment]
Also, I second the suggestion to post up to the discourse, and in particular perhaps people from the Jupyter in HPC workshop this last week in Berkeley might be well-suited to addressing the tutorial.

**Comment by choldgraf** [comment]
@jasongrout I agree re: HPC workshop people - I think we should use these tutorials etc as an opportunity to bring more jovyans into the jupyter community (that is, assuming we're able to find somebody willing to do it :-) )

**Comment by zonca** [comment]
thanks, I posted on discourse as well https://discourse.jupyter.org/t/looking-for-an-instructor-to-teach-jupyterlab-extension-tutorial-in-san-diego-sep-23rd/1373

**Comment by zonca** [comment]
I got no replies on discourse. Would any of you have an update on their availability to teach the JupyterLab extension tutorial at Gateways 2019 in San Diego on September 23rd?

**Comment by choldgraf** [comment]
Hmm - I can ask around at the JupyterLab sprint tomorrow and see if anybody is interested.

One thing - looking at your first post, I'm not actually sure what you are asking somebody to do. Is there already a pre-existing interface for gateways in jupyterlab, and you just want someone to show how it works? Or do you want the SciPy tutorial that @jasongrout linked above?

**Comment by zonca** [comment]
The scipy tutorial

**Comment by Carreau** [comment]
So these dates in September are hard; There are other people traveling; so we need someone to hold the fort at UC Merced. I'll see if I can promote your discourse post on twitter with the Jupyter account.
